qutebrowser: add page (#5850)
tldr-pages/tldr: pages/common/qutebrowser.md

# qutebrowser

> > A keyboard-driven, vim-like browser based on PyQt5.

Reviewer: I can't tell if I like this or not.  At first glance, the sentences seems a bit jumbled without the comma.  Two more seconds of looking at it though and it makes sense.  🤔🤔 
```suggestion
> A keyboard-driven, vim-like browser, based on PyQt5.
```

Author: It's the literal description given by the project, so I just pasted it over. But like you said, read it twice it works better than with the second comma.

> More information: <https://qutebrowser.org/>.

> - Open qutebrowser with a specified (non-default) storage directory:
> 
> `qutebrowser --basedir {{base_directory}}`

Reviewer: "non-default" doesn't seem necessary to add here. Is it?
```suggestion
- Open qutebrowser with a specified storage directory:

`qutebrowser --basedir {{path/to/directory}}`
```
Also, `base_directory` in the variable name doesn't seem to give it much more value than with omitting it 🤔

- Open a qutebrowser instance with temporary settings:

> `qutebrowser --set {{option}} {{value}}`

Reviewer: Could you add an example of listing the available options?  Without that, this example might not be very useful.

- Restore a named session of a qutebrowser instance:

> `qutebrowser --restore {{session_name}}`

Reviewer: Same here.  How can one access a list of valid session names?

Author: As far as I'm aware of, there is not an argument to list (at time of writing, might open a PR there..) saved named sessions. These can be found in the browser itself (`:session-load {tab}`) via the session loading command.

> - Choose how to open URLs if there is already a qutebrowser instance running:

Reviewer: Is this correct?  I'm trying to avoid using "choose" because that sounds slightly ambiguous.
```suggestion
- Launch qutebrowser, opening all links using the specified method:
```

Author: Why `links` instead of `URLs`, just a question.

Reviewer: No particular reason.  I didn't even mean to do that.  Changed back.

`qutebrowser --target {{auto|tab|tab-bg|tab-silent|tab-bg-silent|window|private-window}}`

> - Open qutebrowser with a temporary base directory with stdout logging in json format:

Reviewer: ```suggestion
- Open qutebrowser with a temporary base directory and print logs to stdout as JSON:
```
